compact/far_forward/beampipe_hadron_B0.xml

```diff
@@ -10,7 +10,7 @@
 
   <detectors>
 
-    <detector id="BeamPipeB0_ID" name="BeamPipeB0" type="hadronDownstreamBeamPipe" vis="BeamPipeVis">
```

Comment:
Remove `src/hadronDownstreamBeamPipe.cpp` if now unused.

Reply:
I would like to keep it for the time being since we still have more updates in the coming months, and this particular code is the reference we have used for a while. Once we are sure the new design is set, we can remove it.